src/obr/cli.py

```diff
@@ -133,6 +145,15 @@ def run(ctx, **kwargs):
         os.chdir(kwargs["folder"])
 
     project = OpenFOAMProject().init_project()
+
+    if "--help" == kwargs.get("operations"):
+        project.print_operations()
+        return
```

Comment:
```suggestion
    if  not kwargs.get("operations"):
        project.print_operations()
        return
```
I would prefer it to exit if no operation are specified. (Or if operations are specified which are not in `project.operations`)